Makefile

```diff
@@ -75,5 +72,7 @@ bitstreamData.json: bitstream.bits bithtml.py grid/.dir frames/.dir
 		--bits=bitstream.bits \
 		--dump-grid=$@ --grid-dir=$$PWD/grid
 
+all:: bitstream.html
+
```

Comment:
I can't see any point in generating `bitstream.html` by default. It's not used anymore and it was left just for reference. In fact it should be removed (with the corresponding code in `bithtml.py`) or cleaned and left as optional target.

Reply:
I changed the file to not have bitstream.html build by default